src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { TodoWithUser } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
```

Comment:
Consider adding a more descriptive name for the Props type, like `TodoInfoProps`. 🚀